contrib/cmd/seccompagent/gen-seccomp-example-cfg.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# Detect if we are running inside bats (i.e. inside integration tests) or just
+# called by an end-user
+# bats-core v1.2.1 defines BATS_RUN_TMPDIR
+if [ -z "$BATS_RUN_TMPDIR" ]; then
+	# When not running in bats, we create the config.json
+	set -e
+	runc spec
+fi
+
+# We can't source $(dirname $0)/../../../tests/integration/helpers.bash as that
+# exits when not running inside bats. We can do hacks, but just to redefine
+# update_config() seems clearer. We don't even really need to keep them in sync.
+function update_config() {
+        jq "$1" "./config.json" | awk 'BEGIN{RS="";getline<"-";print>ARGV[1]}' "./config.json"
+}
+
+update_config '.linux.seccomp = {
+                        "defaultAction": "SCMP_ACT_ALLOW",
+                        "listenerPath": "/run/seccomp-agent.socket",
+                        "listenerMetadata": "foo",
+                        "architectures": [ "SCMP_ARCH_X86", "SCMP_ARCH_X32" ],
```

Comment:
So, your first commit adds `SCMP_ARCH_X86_64` to config.json, and this one removes config.json and adds a way to generate it, but it omits `SCMP_ARCH_X86_64`.

What is the purpose of the first commit then?

Reply:
heh, you missed this comment: https://github.com/opencontainers/runc/pull/3206#issuecomment-916297244. It was a typo, fixed now

Reply:
Huh, I thought that the typo was "x89-64", but it was another typo :))))